public/js/app/components/statistics/domain/schools.js

```diff
@@ -4,9 +4,10 @@ const SCHOOLS = {
   EECS: 'EECS',
   ITM: 'ITM',
   SCI: 'SCI',
+  ALLS: 'ALLS',
```

Comment:
It is not clear what is it for a new person. Is it a school with a name "ALLS"? There is already existing label "allSchools" and it has to be reused. We don't want to have different labels meaning the same thing

Reply:
Here I used ALLS because some how allSchools is to long and will cose that bar for the allSchools will be pushed to right of allSchools tick.
![image](https://user-images.githubusercontent.com/18121494/224056977-1b464859-a907-4ea4-8c27-e1fad6176300.png)

 Further we need tick lable in the format from Figma like"All schools" in 2 rows and I can't have "All schools" as a key in data object wich we use for tick lable name. I have to format it within VictoryAxis with tickFormat attribute, then I get right format and I need here l. 
![image](https://user-images.githubusercontent.com/18121494/224071102-964889d5-aef8-4a17-9250-28bde9f2109c.png)
But still bar for the allSchools will be pushed to right because "allSchools" in ORDERED_SCHOOLS array is to long and thats way I used ALLS. Is it any way to make within VictoryAxis brake line of long names?
If I have "ALLS" it will be right?
![image](https://user-images.githubusercontent.com/18121494/224086288-7cb64aee-af24-4078-9085-3cce9a530040.png)

Reply:
It is still has to be refactored. If you reuse all code the only this left will be to divide "All schools" by using f.e., `replace` function where you replace the space symbol with `\n` or by splitting and joining it, f.e. `split(" ").join("\n")` So try to make this solution a long-term solution which will be easy understood by new developers in future

Reply:
About having "All schools" as a key in an object, which I think is not needed here, but anyway I will add a link to read for information. https://stackoverflow.com/questions/8317982/how-can-i-access-a-javascript-object-which-has-spaces-in-the-objects-key

Reply:
check it again